salt/utils/platform.py

```diff
@@ -231,6 +231,14 @@ def is_aarch64():
     return platform.machine().startswith("aarch64")
 
 
+@real_memoize
+def is_arm64():
```

Comment:
Not really sure about adding this new method instead of modifying the existing `aarch64` with something like this:

```py
return re.search(r'^(aarch|arm)64', platform.machine()) is not None
```

I think my initial approach is more conservative, since I'm not modifying the current behavior of the `is_aarch64` method. But essentially, Aarch64 and ARM64 are two different ways to refer to the [same thing](https://en.wikipedia.org/wiki/AArch64).

Aarch64 is how the LLVM community refers to this architecture while Apple calls it ARM64.

Reply:
I would like to ask the opinion of @s0undt3ch and @krionbsd. Since @krionbsd implemented the original `is_aarch64` method and @s0undt3ch is one of the main developers for macOS support.